app/src/main/java/com/mobileproto/hireddit/hireddit/visuals/SpeechFragment.java

```diff
@@ -130,6 +130,7 @@ public void doListen() {
         Log.d(DEBUG_TAG, "Start listening");
         isListening = true;
         updateListeningIndicator();
+        mListener.speak("");
```

Comment:
is this overwriting what's being spoken? isn't there the `stop()` method that we added to the text-to-speech file we can use instead?